bin/plot_krun_results

```diff
@@ -698,24 +708,13 @@ class ProcessExecChart(object):
             self._plot_outliers(inset, large_markers=False)
             self._plot_changepoints(inset, large_markers=False)
             pyplot.draw()  # Force xticklabels to be drawn.
-            if self.x_bounds[0] > 0:
```

Comment:
On yay2.pdf, the x-ticklabels on the wallclock data chart have become floats, and the xticklabels on the inset don't seem to include the first and last datum.